runtime/util/vmargs.c

```diff
@@ -626,6 +625,12 @@ optionValueOperations(J9PortLibrary *portLibrary, J9VMInitArgs* j9vm_args, IDATA
 						default:
 							return OPTION_MALFORMED;
 					}
+
+					if (action==GET_MEM_VALUE) {
```

Comment:
Pls add spaces around `==`.

Reply:
I assume both of these hold only for the lines I'm touching? I originally went with this formatting to maintain consistency with the existing code.